aisynphys/connectivity.py

```diff
@@ -323,6 +323,8 @@ def fit(cls, x, conn, init=(0.1, 150e-6), bounds=((0.001, 1), (10e-6, 1e-3)), fi
         n = 6
         p_bins = np.linspace(bounds[0][0], bounds[0][1], n)
         s_bins = np.linspace(bounds[1][0], bounds[1][1], n)
+        if fixed_size is not None:
+            s_bins = [fixed_size] * n
```

Comment:
I think this doesn't belong.. the `fixed_size` arg got committed here by mistake.

Reply:
There are a few things like that....I don't quite know what to do about it since it seems like it's because the db17 branch is behind master